### DISCUSSION TITLE
Corrupted WSL

### DISCUSSION THREAD

**Discussion by cqundefine** [microsoft/WSL#7025] [Q&A]
So I was converting WSL from version 1 to version 2, and my power went out and now WSL just doesn't work and I want to ask if there is any way that I can recover my files saved inside of it.

**Comment by therealkenc** [comment]
It _might_ be possible to fish files out of `%LocalAppData%\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\rootfs\home` from the Windows side, if it still exists. The exact path will vary depending on the distribution, buy will stand out in a directory listing.
